bagit.xml

```diff
@@ -699,25 +724,32 @@ of a checksum are outside the scope of this document.
         <t>
 The name of the checksum algorithm &must; be normalized for use in the
 manifest's filename by lowercasing the common name of the algorithm and
-removing all non-alphanumeric characters.
-</t>
+removing all non-alphanumeric characters. Following is a partial list
+mapping common algorithm names to normalized names:
+          <list style="symbols">
+            <t>MD-5: md5</t>
+            <t>SHA-1: sha1</t>
+            <t>SHA-256: sha256</t>
+            <t>SHA-512: sha512</t>
+
+</list></t>
         <t>
-  Bag creation and validation tools &must; support the SHA-2 family of
+  For BagIt 1.0, bag creation and validation tools &must; support the SHA-2 family of
```

Comment:
We probably shouldn't litter this document with the version because it will be much harder to find them all later when the next version comes out. Perhaps you were thinking _Starting with_ instead of _For_?

Reply:
We discussed this on the call – the idea was that we want to say that existing 0.97 implementations aren't immediately invalid.

Reply:
That is why I think it reads better as "Starting with Bagit 1.0, bag..."

Reply:
Agreed: the point was that the version number wasn't actually a problem: if we wanted it to change we could use `&current-bagit-version;`. Tweaking the wording to note that this is a backwards-compatibility allowance makes sense.

Reply:
Changing.